docs/streaming.mdx

```diff
@@ -23,53 +23,62 @@ This is the format of the `unprocessedRecords` table:
+4. Any synced table has a field called `id`, which syncs with the Salesforce ID field.
 
 ## The streaming sync method, step-by-step
 
 At cadences as fast as every 30 seconds, Bracket performs the following:
 
 <Steps>
   <Step title="First Step">
-    Bracket checks the Salesforce pub/sub API to see if any events have occurred since the last successful data read/write. E.g., if there have been no changes to your Salesforce object in six minutes, Bracket is checking the pub/sub API for any events from the last six minutes.
-
-    If there are no new events, and it's been less than 10 minutes since the last successful poll, Bracket does not poll Salesforce.
+    Bracket checks the Salesforce pub/sub API to see if any events have occurred since the last
+    successful data read/write. E.g., If the last change read by Bracket on your Salesforce
+    object was processed six minutes ago, Bracket will only check the pub/sub API for any events
+    with a `commitTimestamp` greater than the last six minutes.
+
+    If there are no new events, and it's been less than 10 minutes since the last successful
+    poll, Bracket does not poll Salesforce--this is where Brackets saves you from costly REST
+    API usage.
   </Step>
   <Step title="Second Step">
-    If there are events reported by the pub/sub API, _or_ if it's been greater than 10 minutes since the last poll, Bracket polls Salesforce via REST API for all records modified or created since the last successful poll. We store these records in `unprocessedRecords`.
+    If there are events reported by the pub/sub API, _or_ if it's been greater than 10 minutes since the last poll, Bracket polls Salesforce via REST API for all records modified or created since the last successful poll. We store these records in `unprocessed_records`.
 
-    Consistent 10-minute polling intervals make sure any events missed by pub/sub are still ultimately synced.  
+    Consistent 10-minute polling intervals make sure any events missed by pub/sub are still ultimately synced.
   </Step>
   <Step title="Third Step">
-    Bracket reads records from the Postgres trigger log that have `status='unprocessed'` and queries records with the corresponding primary keys from the synced Postgres table. 
+    Bracket reads records from the Postgres trigger log that have `status='UNPROCESSED'` and queries records with the corresponding primary keys from the synced Postgres table.
   </Step>
   <Step title="Fourth Step">
-     Bracket writes Salesforce edits to Postgres. The record is then marked as `PROCESSED` or `ERRORED` in `unprocessedRecords`. If `ERRORED`, Bracket will try syncing the erroneous record at the next poll attempt.
+    Bracket writes Salesforce edits to Postgres. The record is then marked as `PROCESSED` or
+    `ERROR` in `unprocessed_records`. If `ERRORED`, Bracket will try syncing the erroneous
+    record at the next poll attempt only if a new change has been detected.
```

Comment:
but the poll only happens if there's a change or it's been 10 minutes, right? e.g. if it's just the ten minute mark, then we'll still retry the errors. So it's not totally accurate to say "only if a new change has been detected"